app/(afterLogin)/(okrplanning)/okr/_components/keyresultView/numericView/index.tsx

```
@@ -47,6 +47,7 @@ const NumericView: React.FC<OKRProps> = ({ keyValue, index, isEdit, form }) => {
 
   const { isMobile } = useIsMobile();
 
+
   return (
```

ESLint

[error] Delete `⏎`

(prettier/prettier)